examples/pytorch/gat/main.py

```diff
@@ -133,17 +118,46 @@ def transform_batch(idx: list) -> dict:
 
 
 def _main():
-    # setup default logging component.
     setup_default_logging_config(enable_telemetry=True)
-
     ray.init(num_cpus=4)
 
-    args = get_args(init_args)
+    trainer = HorovodTrainer(
+        train_func,
+        train_loop_config={
+            "data_dir": "/tmp/cora",
+            "model_dir": "/tmp/model_output",
+            "sample_file": "/tmp/cora/train.nodes",
+            "partitions": 1,
+            "num_epochs": 180,
+            "batch_size": 140,
+            "feature_idx": 0,
+            "feature_dim": 1433,
+            "label_idx": 1,
+            "label_dim": 1,
+            "num_classes": 7,
+            "mode": "train",
+        },
+        scaling_config=ScalingConfig(num_workers=1),
+    )
+    trainer.fit()
 
     trainer = HorovodTrainer(
         train_func,
-        train_loop_config={"args": args},
-        scaling_config=ScalingConfig(num_workers=1, use_gpu=args.gpu),
+        train_loop_config={
+            "data_dir": "/tmp/cora",
+            "model_dir": "/tmp/model_output",
+            "sample_file": "/tmp/cora/test.nodes",
+            "partitions": 1,
+            "num_epochs": 1,
+            "batch_size": 1000,
+            "feature_idx": 0,
+            "feature_dim": 1433,
+            "label_idx": 1,
+            "label_dim": 1,
+            "num_classes": 7,
+            "mode": "evaluate",
```

Comment:
Evaluate in a test or validation dataset directly in the training loop instead. See the pyg example in Ray.